word_parser.rb

```diff
@@ -0,0 +1,52 @@
+module WordParser
+  def self.read_file(file_path)
+    File.open(file_path) { |f| f.read }
+  end
+
+  def self.to_arrayed_words(string)
+    string.gsub!(/\s\-\s/, ' ')
```

Comment:
I thought about that, but did it here because it's a form of word separation, like whitespace and not an actual word. I don't remove hyphens inside of words and this was a fix when I noticed I had a word that was just a hyphen when I ran the program.